package.json

```diff
@@ -10,9 +10,17 @@
     "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
     "test": "npm run unit",
     "lint": "eslint --ext .js,.vue src test/unit/specs",
-    "start": "node server.js"
+    "start": "node server.js",
+    "postinstall": "npm run updatesemantic",
+    "updatesemantic": "cp ./src/config/css/theme.config ./node_modules/semantic-ui-less/theme.config ; cp ./src/config/css/defaultsite.variables ./node_modules/semantic-ui-less/themes/default/globals/site.variables"
```

Comment:
Nice, everything looks good. After sorting the Windows command line issue we can merge.